recipes/r-xslt/meta.yaml

```
@@ -0,0 +1,90 @@
+{% set version = '1.4.4' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-xslt
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/xslt_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/xslt/xslt_{{ version }}.tar.gz
+  sha256: 64aacaff2dacb474c3abe2e91589676f8741981011ca7fc35b814e0ceb884c31
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}sed               # [win]
+    - {{ posix }}grep              # [win]
+    - {{ posix }}autoconf
+    - {{ posix }}automake          # [not win]
+    - {{ posix }}automake-wrapper  # [win]
+    - {{ posix }}pkg-config
+    - {{ posix }}make
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - libxslt
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-rcpp
+    - libxslt
+    - r-xml2 >=1.3.0
```

This package itself links against `libxml2`, so we need to declare that. Also, let's reorder:

```suggestion
    - r-xml2 >=1.3.0
    - libxml2
    - libxslt
```